src/v2/Apps/Gene/Components/GeneArtworkFilter.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import { createRefetchContainer, RelayRefetchProp, graphql } from "react-relay"
+import { useRouter } from "v2/Artsy/Router/useRouter"
+import { BaseArtworkFilter } from "v2/Components/v2/ArtworkFilter"
+import { ArtworkFilterContextProvider } from "v2/Components/v2/ArtworkFilter/ArtworkFilterContext"
+import { GeneArtworkFilter_gene } from "v2/__generated__/GeneArtworkFilter_gene.graphql"
+
+interface GeneArtworkFilterProps {
+  gene: GeneArtworkFilter_gene
+  relay: RelayRefetchProp
+}
+
+const GeneArtworkFilter: React.FC<GeneArtworkFilterProps> = ({
+  gene,
+  relay,
+}) => {
+  const { match } = useRouter()
```

Comment:
Not sure why I have to pass this/why the provider doesn't just set this as the default.

Reply:
Meaning, what's passed into the `filters` prop below? I can imagine it being set by default with the option to override with non-location.query state.